libs/Onboarding/tests/OnboardingTestHelpers/Constants.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <string>
+
+namespace Status::Testing::Constants {
+
+constexpr auto userDataDirName = "StatusTest";
+constexpr auto statusGoDataDirName = "data";
+constexpr auto tmpDataDirName = "tmp";
+constexpr auto logsDataDirName = "logs";
+constexpr auto qtDataDirName = "qt";
+constexpr auto keystoreDataDirName = "keystore";
+constexpr auto globalSettingsFileName = "global";
```

Comment:
inline

Reply:
Changed it for now. However, these constants are not required as global variables and should be refactored moving forward.

Reply:
- If they are defined in the header file they should be defined as inline to leverage C++17 inline variables' external linkage
- If they are not considered global variables then should be defined as an implementation detail in the source file under an unnamed namespace